modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java

```diff
@@ -1179,6 +1181,34 @@ private ObjectDefinition _publishObjectDefinition(
 								_createObjectFieldSetting(
 									"maximumFileSize", "100"))
 						).build(),
+						new AutoIncrementObjectFieldBuilder(
+						).labelMap(
+							LocalizedMapUtil.getLocalizedMap(
+								RandomTestUtil.randomString())
+						).name(
+							"testAutoIncrementField"
+						).objectFieldSettings(
+							Arrays.asList(
+								new ObjectFieldSettingBuilder(
```

Comment:
I see a pattern of using `_createObjectFieldSetting` instead

Reply:
I will update it, I just followed the Object Services pattern